packages/react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -68,6 +75,18 @@ export class DetailsColumnBase extends React.Component<IDetailsColumnProps> {
     const classNames = this._classNames;
     const IconComponent = useFastIcons ? FontIcon : Icon;
 
+    const defaultOnRenderHeader = (): JSX.Element | null => {
```

Comment:
Define this outside the function at module-level, and have it take in `props: IDetailsColumnProps`. An override should be allowed to mess with the props to `defaultRender` to change the behavior, too.

Reply:
Should defaultOnRenderHeader be a public function if it's at the module level?

Reply:
Updated